<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>8</horstretch>
    <verstretch>8</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QPushButton {
        background-color: #D11A0B;
        color: white;
        border-radius: 20px;
        padding: 0px;
        border: 1px solid #D11A0B;
        font-size: 12px;
    }
    QPushButton:hover {
        background-color: #FFFFFF;
        color: #D11A0B;
    }
    QPushButton:pressed {
        background-color: #FFFFFF;
        color: #D11A0B;
    }
    QTableWidget {
        background-color: #FFFFFF;
        border: 1px solid #dcdcdc;
        gridline-color: #dcdcdc;
    }
    QTableWidget QHeaderView::section {
        background-color: #D11A0B;
        color: #FFFFFF;
        padding: 5px;
        border: 1px solid #D11A0B;
    }
   </string>
  </property>
  <widget class="QPushButton" name="ButtonAccueil">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string/>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonGestion">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gestion des accès</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPerimetres">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Périmètres</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonZones">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Identification zones</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gestion des accès</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: gray;
	color: #ffffff; /* Couleur du texte (blanc dans cet exemple) */
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>Listes des accès</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetAcces">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>201</y>
     <width>521</width>
     <height>491</height>
    </rect>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prénom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mail</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Acces périmètre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bâtiment</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEditNom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrenom">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>710</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>710</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>690</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Prénom</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>690</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mail</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>690</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Accès périmètre</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetNouv">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prénom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mot de passe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mail</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidgetModule">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Module</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hygiène</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Qualité</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>RSE</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ButtonEntrer">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>740</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Entrer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>690</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bâtiment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSupprimer">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>740</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAcces">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>710</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxBatiment">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>710</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Accueil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bâtiment B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bâtiment C</string>
    </property>
   </item>
  </widget>
  <zorder>ButtonAccueil</zorder>
  <zorder>ButtonGestion</zorder>
  <zorder>ButtonPerimetres</zorder>
  <zorder>ButtonZones</zorder>
  <zorder>label</zorder>
  <zorder>tableWidgetAcces</zorder>
  <zorder>lineEditNom</zorder>
  <zorder>lineEditPrenom</zorder>
  <zorder>lineEditMail</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>tableWidgetNouv</zorder>
  <zorder>tableWidgetModule</zorder>
  <zorder>ButtonEntrer</zorder>
  <zorder>label_2</zorder>
  <zorder>label_7</zorder>
  <zorder>ButtonSupprimer</zorder>
  <zorder>comboBoxAcces</zorder>
  <zorder>comboBoxBatiment</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
